src/security/ServerOptions.cc

```diff
@@ -264,13 +266,13 @@ Security::ServerOptions::createStaticServerContext(AnyP::PortCfg &port)
         }
 #endif
 
+        if (!loadClientCaFile())
```

Comment:
If this call must go before updateContextConfig() for some reason, we should at least add a corresponding comment here.

Reply:
done